public/locales/ar/translation.json

```diff
@@ -8,6 +8,309 @@
+      },
+      {
+        "title": "المصلحة الاجتماعية",
+        "name": "جيني ويلسون",
+        "portraitImg": "doctorImg",
+        "phoneNumber": "+1-340-5558-235 ",
+        "address": "٩٠٩٤ شارع باي ميادوز ، مدينة كونري ، ٣٠٠١٢",
+        "moreInfo": "لوريم ايبسوم دولار سيت أميت ,كونسيكتيتور أدايبا يسكينج أليايت,سيت دو أيوسمود تيمبور أنكايديديونتيوت لابوري ات دولار ماجنا أليكيوا . يوت انيم أد مينيم"
+      }
+    ],
+    "jobOpportunities": [
+      {
+        "title": "طباخ",
+        "name": "شركة داميون بيتيرسون",
+        "description": "لوريم ايبسوم دولار سيت أميت ,كونسيكتيتور أدايبا يسكينج أليايت,سيت دو أيوسمود تيمبور أنكايديديونتيوت لابوري ات دولار ماجنا أليكيوا . يوت انيم أد مينيم",
+        "address": "اسطنبول، تركيا",
+        "phoneNumber": "+906467291373",
+        "postingDate": "27/10/2020",
+        "id": 0
+      },
+      {
+        "title": "مصمم مواقع",
+        "name": "شركة داميون بيتيرسون",
+        "description": "لوريم ايبسوم دولار سيت أميت ,كونسيكتيتور أدايبا يسكينج أليايت,سيت دو أيوسمود تيمبور أنكايديديونتيوت لابوري ات دولار ماجنا أليكيوا . يوت انيم أد مينيم",
+        "address": "اسطنبول، تركيا",
+        "phoneNumber": "+906467291373",
+        "postingDate": "27/10/2020",
+        "id": 1
+      },
+      {
+        "title": "مترجم",
+        "name": "شركة داميون بيتيرسون",
+        "description": "لوريم ايبسوم دولار سيت أميت ,كونسيكتيتور أدايبا يسكينج أليايت,سيت دو أيوسمود تيمبور أنكايديديونتيوت لابوري ات دولار ماجنا أليكيوا . يوت انيم أد مينيم",
+        "address": "اسطنبول، تركيا",
+        "phoneNumber": "+906467291373",
+        "postingDate": "27/10/2020",
+        "id": 2
+      },
+      {
+        "title": "مهندس معماري",
+        "name": "شركة داميون بيتيرسون",
+        "description": "لوريم ايبسوم دولار سيت أميت ,كونسيكتيتور أدايبا يسكينج أليايت,سيت دو أيوسمود تيمبور أنكايديديونتيوت لابوري ات دولار ماجنا أليكيوا . يوت انيم أد مينيم",
+        "address": "اسطنبول، تركيا",
+        "phoneNumber": "+906467291373",
+        "postingDate": "27/10/2020",
+        "id": 3
+      },
+      {
+        "title": "مدير علاقات عامة",
+        "name": "شركة داميون بيتيرسون",
+        "description": "لوريم ايبسوم دولار سيت أميت ,كونسيكتيتور أدايبا يسكينج أليايت,سيت دو أيوسمود تيمبور أنكايديديونتيوت لابوري ات دولار ماجنا أليكيوا . يوت انيم أد مينيم",
+        "address": "اسطنبول، تركيا",
+        "phoneNumber": "+906467291373",
+        "postingDate": "27/10/2020",
+        "id": 4
+      },
+      {
+        "title": "أستاذ",
+        "name": "شركة داميون بيتيرسون",
+        "description": "لوريم ايبسوم دولار سيت أميت ,كونسيكتيتور أدايبا يسكينج أليايت,سيت دو أيوسمود تيمبور أنكايديديونتيوت لابوري ات دولار ماجنا أليكيوا . يوت انيم أد مينيم",
+        "address": "اسطنبول، تركيا",
+        "phoneNumber": "+906467291373",
+        "postingDate": "27/10/2020",
+        "id": 5
```

Comment:
whys this id field here, can you just use the title? fix in separate pr

Reply:
ok i will